### ISSUE TITLE
change event not firing when value changes

### ISSUE THREAD

**Issue by stijnvanraes** [igorescobar/jQuery-Mask-Plugin#325]
Hi 

When the value of my input field changes, sometimes the change event is not triggering.
I use jquery 2.1.3 and following mask:

```
$('#data1,#date2').mask('00-00-0000');
```

And following simple code to catch the change event

```
$('#data1,#date2')change(function() {
   console.log('change');
});
```

Then when I input 01012015, the mask is applied and the change event triggers.
But then when I change the value to 02-01-2015, the change event does not trigger.

Greetings

**Comment by igorescobar** [comment]
![screen shot 2015-07-15 at 12 03 12 pm](https://cloud.githubusercontent.com/assets/24032/8701545/c59fb5ea-2ae8-11e5-9d92-41cf60d6b3aa.png)

**Comment by stijnvanraes** [comment]
Yes, should have created a fiddle right away. https://jsfiddle.net/fpo43s7g/

Apparently the problem only occurs in chrome and IE, and only when you use tab to go out of the field.
To reproduce input 01012015 into the first field, then tab out -> works fine
Then change it to 02-01-2015 and tab out of the field -> change event doesn't trigger.

**Comment by igorescobar** [comment]
@stijnvanraes could you test this version and see if the problem still occurs?
https://raw.githubusercontent.com/igorescobar/jQuery-Mask-Plugin/master/src/jquery.mask.js

Thanks :+1:

**Comment by furqanms** [comment]
I had same issue I did the following : https://github.com/furqanms/jQuery-Mask-Plugin/commit/c1eddda4b9b5f4f6dec1debb8e21c1d0dc4cd312#commitcomment-12194042

**Comment by stijnvanraes** [comment]
@igorescobar the new version works, thanks a lot! Keep up the great work!

**Comment by igorescobar** [comment]
Awesome @stijnvanraes :+1: thanks for your help!

**Comment by igorescobar** [comment]
This fix can be found in v1.13.2

**Comment by pwinch** [comment]
Love the mask script.  fails in chrome :(
